recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0-3" %}
+
+package:
+    name: r-gsw
+    version: {{ version | replace('-', '_') }}
+
+source:
+    fn: gsw_1.0-3.tar.gz
+    url:
+        - http://cran.r-project.org/src/contrib/gsw_{{ version }}.tar.gz
+        - http://cran.r-project.org/src/contrib/Archive/gsw/gsw_{{ version }}.tar.gz
+    sha256: e24c6802467546d616531499c2a98610fdb5c35c8a1e161b6d6b887c4c814331
+
+build:
+    number: 0
+    script: R CMD INSTALL --build .
```

Comment:
That actually worked. The failure seems to be due to the lack of a `c` compiler.

```shell
C:\Miniconda\conda-bld\work\gsw>R CMD INSTALL --build . 
* installing to library 'C:/Miniconda/envs/_build/R/library'
* installing *source* package 'gsw' ...
** package 'gsw' successfully unpacked and MD5 sums checked
** libs
Warning: running command 'make -f "C:/Miniconda/envs/_build/R/etc/i386/Makeconf" -f "C:/Miniconda/envs/_build/R/share/make/winshlib.mk" SHLIB="gsw.dll" OBJECTS="gsw_oceanographic_toolbox.o gsw_p_from_z.o gsw_saar.o wrappers.o"' had status 127
ERROR: compilation failed for package 'gsw'
* removing 'C:/Miniconda/envs/_build/R/library/gsw'
Command failed: C:\Windows\system32\cmd.exe /c call bld.bat
Command exited with code 1
```